src/climate_indices/__main__.py

```diff
@@ -719,7 +721,22 @@ def _compute_write_index(keyword_arguments):
             chunks = {"time": -1}
         else:
             raise ValueError(f"Invalid 'input_type' keyword argument: {input_type}")
-    dataset = xr.open_mfdataset(files, chunks=chunks)
+    # Since multiple variables can be in the same file, de-duplicate the filelist.
+    dataset = xr.open_mfdataset(list(set(files)), chunks=chunks)
+    output_chunksizes = {}
+    if keyword_arguments['chunksizes'] == 'input':
+        # Find the first variable with chunksizes set and use that
+        # Note that the netcdf spec doesn't require that all data variables
+        # have the same chunk sizes.
+        for da in dataset.data_vars.values():
+            if not da.encoding.get("contiguous", True):
+                # tuple of chunksizes, respectively by dimension
+                output_chunksizes = da.encoding.get("chunksizes", ())
+            if output_chunksizes:
+                break
+
+    # Now apply the requested chunks
+    dataset = dataset.chunk(chunks)
```

Comment:
Actually, this isn't needed since the file(s) are loaded with the specified chunks.